arcgis-ios-sdk-samples/Display information/Display grid/DisplayGridSettingsViewController.swift

```diff
@@ -265,4 +257,10 @@ class DisplayGridSettingsViewController: UIViewController, HorizontalPickerDeleg
             changeLabel(color: selectedColor)
         }
     }
+    
+    // MARK: UITableViewDelegate
+    
+    override func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
+        return false
```

Comment:
It might be easier to just disable selection on the table view for now:

![screen shot 2018-10-12 at 4 08 51 pm](https://user-images.githubusercontent.com/2257493/46897710-30ae7f00-ce39-11e8-9684-2e584106b573.png)

When the horizontal pickers go away, you could then re-enable selection and update the selection style to `None` for cells that should not be selectable.

![screen shot 2018-10-12 at 4 10 53 pm](https://user-images.githubusercontent.com/2257493/46897739-753a1a80-ce39-11e8-9ffa-bca26aedc222.png)